### ISSUE TITLE
[Content styleguide] Content doesn't align with style guidance in VA.gov content style guide. (05.08.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#28327]
#### VFS team name
Healthcare Experience

#### VFS product name
Check-in MVP

#### Point of Contact/Reviewers
Peggy Gannon

---

### Platform Issue
Content doesn't align with style guidance in VA.gov content style guide.

### Issue Details
Section heads and labels should be sentence cased

### Supporting Guidance
[Category 5, Issue 8](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

---

### Platform Recommendation
Please render the Need help? header in sentence case > Need help? 

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by jparker0425** [comment]
Will address and modify to sentence case post MVP.

**Comment by mdewey** [comment]
This should be resolve in staging.

**Comment by shiragoodman** [comment]
adding labels so ticket is consistent with post-pilot process
